yarn.lock

```diff
@@ -2919,27 +2919,6 @@ algoliasearch@4.8.2:
     "@algolia/requester-node-http" "4.8.2"
     "@algolia/transporter" "4.8.2"
 
-algoliasearch@^3.35.1:
```

Comment:
unfortunately you need to keep this entry in the lock file, otherwise algoliasearch-v3 won't build anymore in storybook as far as I can tell (it doesn't deal properly with the aliased module + places.js having a dependency

Reply:
Not sure why it got deleted.

Reply:
Yarn thinks this is the right resolution (since it deduplicates), but some other tools try to use the node_modules folder name, which is `algoliasearch-v3` if it gets deduplicated